style/js/shield_defs.js

```diff
@@ -2159,6 +2159,52 @@ export function loadShields(shieldImages) {
     null
   );
 
+  // Sweden
+  [
+    "RV", // Riksväg - national road
+    "LV", // Länsväg - primary county road
+    "AB", // Individual counties
+    "C",
+    "D",
+    "E",
+    "F",
+    "G",
+    "H",
+    "I",
+    "K",
+    "M",
+    "N",
+    "O",
+    "S",
+    "T",
+    "U",
+    "W",
+    "X",
+    "Y",
+    "Z",
+    "AC",
+    "BD",
```

Comment:
(Or `SE:county`, following [the more common convention](https://wiki.openstreetmap.org/wiki/Key:network#Basic_format) for countries that don’t make a lot of local distinctions.)

Reply:
As a point of reference, a lot of U.S. states also have county route networks that can repeat numbers across county lines and use a uniform shield. That shield typically bears the county name, and people even call them “Abc County Route 123” sometimes. But the `network`s got simplified to omit this detail because most renderers don’t need that extra context and a spatial query can already provide it if necessary. Geocoders would also find these county-qualified `network`s redundant to the containing boundary information they already provide. Another kind of `network` consumer is editors, but ideally editors would understand secondary tags like `is_in:county` or (hypothetically) `disambiguator` to label routes less confusingly in a list of relations.

Fortunately, the list of counties in Sweden is pretty manageable and hopefully stable. By contrast, we should probably stop rendering `network`s for China’s county, district, and village routes, because there’s no reasonable way to maintain that list over time as it grows and changes: https://github.com/ZeLonewolf/openstreetmap-americana/pull/213#discussion_r820214498.

Reply:
Yes I saw that explanation.  If the tagging already had significant use and any other data consumer was already using it I wouldn't question it.  I just want to make sure we as style authors don't end up promoting a needlessly complex tagging scheme that no data consumer actually _does_ find useful.  Without input from any other Swedish mappers it just makes me a bit uncomfortable to be the first to adopt the new scheme.

Reply:
In the US we've gone with nomenclature that matches how the network is abbreviated in customary use.  For example, our Interstate highways are abbreviated as "I" so "Interstate 95" is normally described verbally as "I-95" so hence `network=US:I`.

If you're not getting any pushback and you've done due diligence in notifying the community then I don't see much issue in going along with what you've decided.